lib/src/steps/include_test_step.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:automated_testing_framework/automated_testing_framework.dart';
+import 'package:meta/meta.dart';
+
+/// Test step that allows to execute all the steps from another [Test].
+class IncludeTestStep extends TestRunnerStep {
+  IncludeTestStep({
+    this.suiteName,
+    @required this.testName,
+  }) : assert(testName != null);
+
+  final String suiteName;
+  final String testName;
+
+  /// Creates an instance from a JSON-like map structure.  This expects the
+  /// following format:
+  ///
+  /// ```json
+  /// {
+  ///   "suiteName": <String>,
+  ///   "testName": <String>
+  /// }
+  /// ```
+  ///
+  /// See also:
+  /// * [TestStep.fromDynamic]
+  static IncludeTestStep fromDynamic(dynamic map) {
+    IncludeTestStep result;
+
+    if (map != null) {
+      result = IncludeTestStep(
+        suiteName: map['suiteName'],
+        testName: map['testName'],
+      );
+    }
+
+    return result;
+  }
+
+  /// Executes the step. This will first load all the tests through the
+  /// [TestController] and then will filter the first [PendingTest] with the
+  /// same [testName]. If there is at least one [PendingTest] that fit, this
+  /// will iterate through the test's list of steps and will await the
+  /// execution of each one. If not, this will throw an [Exception], failing the
+  /// step.
+  @override
+  Future<void> execute({
+    TestReport report,
+    TestController tester,
+  }) async {
+    log(
+      "include_test('$testName', '$suiteName')",
+      tester: tester,
+    );
+
+    var suiteTests = await tester.loadTests(
```

Comment:
Unfortunately, this logic is a bit backwards.  Since the framework was originally released without suite names as an option, it's likely suite names are `null`.  But test names are required.  So you need to load all pending tests then find the appropriate one as opposed to loading the ones for the current suite.

The good news is that `PendingTest`'s are intended to be "lightweight" in that loading them is cheap from an app perspective.  Loading them all shouldn't need to be avoided, and filtering them all should be cheap too.

Reply:
My bad, I read this wrong, this is good as-is.